python/feldera/sql_context.py

```diff
@@ -266,7 +266,7 @@ def input_pandas(self, table_name: str, df: pandas.DataFrame, force: bool = Fals
 
         tbl = self.tables.get(table_name)
 
-        if tbl is None:
```

Comment:
This redundancy between Python and SQL is not good, the Python code should get the list of registered tables from the compiler. Why doesn't the Python just supply a SQL script?

Reply:
Also, this table will be synthesized only if the now() function is called in the program. Otherwise it won't be there.

Reply:
That's how it will work after the API refactoring.